protocol/avdl/secret_ui.avdl

```diff
@@ -36,4 +36,22 @@ protocol secretUi {
   GetPassphraseRes getKeybasePassphrase(int sessionID, string username, string retry);
 
   string getPaperKeyPassphrase(int sessionID, string username);
+
+  record SecretStorageFeature {
+    boolean allow;
+    string label;
+  }
+
+  record GUIEntryFeatures {
+    SecretStorageFeature secretStorage;
+  }
+
+  record GUIEntryArg {
+    string windowTitle;
+    string prompt;
```

Comment:
Hrr, adding `retryLabel` probably is cleaner, so that we can call attention to it.  Sorry for not getting this right the first time, @patrickxb!